package.json

```
@@ -8,9 +8,11 @@
   },
   "dependencies": {
     "@adobe/aio-cli-lib-app-config": "^0.2.2",
-    "@adobe/aio-lib-console-project-installation": "^2.0.2",
+    "@adobe/aio-cli-lib-console": "^4.0.0",
+    "@adobe/aio-lib-console-project-installation": "^2.0.1",
```

@AmyJZhao , recent changes from **main** have "^2.0.2"
```suggestion
    "@adobe/aio-lib-console-project-installation": "^2.0.2",
```